tests/test_model/test_fnn.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import pytest
+
+from pina.model import FeedForward
+
+
+data = torch.rand((20, 3))
+input_vars = 3
+output_vars = 4
+
+
+def test_constructor():
+    FeedForward(input_vars, output_vars)
+    FeedForward(input_vars, output_vars, inner_size=10, n_layers=20)
+    FeedForward(input_vars, output_vars, layers=[10, 20, 5, 2])
+    FeedForward(input_vars, output_vars, layers=[10, 20, 5, 2],
+                func=torch.nn.ReLU)
+    FeedForward(input_vars, output_vars, layers=[10, 20, 5, 2],
+                func=[torch.nn.ReLU, torch.nn.ReLU, None, torch.nn.Tanh])
+
+
+def test_constructor_wrong():
+    with pytest.raises(RuntimeError):
+        FeedForward(input_vars, output_vars, layers=[10, 20, 5, 2],
+                    func=[torch.nn.ReLU, torch.nn.ReLU])
+
+
+
+def test_forward():
+    dim_in, dim_out = 3, 2
+    fnn = FeedForward(dim_in, dim_out)
+    output_ = fnn(data)
+    assert output_.shape == (data.shape[0], dim_out)
```

Comment:
Could you add also a small test about extra features?

Reply:
Extra features are passed only to `Network` class, in order to maintain  `FeedForward` utilisable outside PINA. Thus, the `pinn` constructor us modified. Now it takes also `extra_feats` argument.

I was missing tests for extrafeatures for `Network` class (added in [17cde2c](https://github.com/mathLab/PINA/pull/88/commits/17cde2c1b277041bfdbf8f6321b313473bfffb0a)), and `PINN` class (added in [b9dfbcd](https://github.com/mathLab/PINA/pull/88/commits/b9dfbcd9cfe3873d64f6cbf187b2696613e22011))

Reply:
Ok ok